staff/jose-rodriguez/precourse/calculadora/index-pro.js

```diff
@@ -0,0 +1,110 @@
+var USER_NUMBERS = [];
```

Comment:
It's better to declare as a const instead of a var, because USER_NUMBERS is not going to mutate

Reply:
thanks for the comment. I didn't use JS ES6 in the precourse